rdmd_test.d

```diff
@@ -675,11 +678,11 @@ void runFallbackTest(string rdmdApp, string buildCompiler, string model)
        if an explicit --compiler flag is not provided, rdmd should
        search its own binary path first when looking for the default
        compiler (determined by the compiler used to build it) */
-    string localDMD = buildPath(tempDir(), baseName(buildCompiler));
-    std.file.write(localDMD, "empty shell");
+    string localDMD = buildPath(tempDir(), baseName(buildCompiler).setExtension(binExt));
```

Comment:
Isn't the build compiler specified in the invocation of the test suite?  Shouldn't the invocation (e.g. in appveyor setup) specify the name correctly rather than relying on the test suite to magically append `binExt`?

Reply:
Well, both "dmd" and "dmd.exe" are valid compiler *commands*, but only the latter is the filename of an executable on Windows.

Reply:
Yes, but this is a slightly different situation from the line in `rdmd` where the fallback compiler path is determined.

I guess another way of putting it is: what happens if the user of the test suite does explicitly specify `--rdmd-default-compiler=dmd.exe`?

Reply:
I think that should be synonymous with `=dmd`, as with `=DMD` or `=DMD.EXE`.

Reply:
OK, cool.  I'd missed that `setExtension` rewrites any existing extension in place.